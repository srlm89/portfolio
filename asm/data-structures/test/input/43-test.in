ERASE actual/43-test.out
INSERT 4 1.019829 Battle for Terra [2007] [1080p]
PRINT_IN_ORDER actual/43-test.out
INSERT 33 2.626349 The Brave One 720P
INSERT 1 4.538350 Shoot.Em.Up.720p.Bluray.x264-SEPTiC
REMOVE_ID 33
PRINT_IN_ORDER actual/43-test.out
INSERT 32 1.955491 Blade 1   720p
INSERT 21 4.854016 High school musical 2006    720p
INSERT 40 9.826899 Robots.WS.AC3.x264.BluRay-PTM   720p
REMOVE_ID 4
PRINT_IN_ORDER actual/43-test.out
INSERT 43 6.240046 The.Warlords.2007.720p.BluRay.DTS.x264-ESiR
INSERT 12 9.975061 High school musical 2006    720p
INSERT 27 1.032175 Bride Wars   720p
INSERT 14 7.914612 Mission Impossible Trilogy   720p
REMOVE_ID 43
REMOVE_ID 14
PRINT_IN_ORDER actual/43-test.out
INSERT 19 5.367680 Mission Impossible Trilogy   720p
INSERT 36 8.260447 Hellboy 1    1080p
INSERT 9 3.118694 Kill Shot   720p
INSERT 45 8.391770 Shark Tale [2004] [1080p]
INSERT 7 5.316481 Terminator.1984.720p.BluRay.x264-ESiR
REMOVE_ID 27
REMOVE_ID 32
PRINT_IN_ORDER actual/43-test.out
INSERT 31 3.895870 Meet The Fockers    720p
INSERT 37 1.272687 SAW IV
INSERT 30 0.589414 Eastern.Promises.2007.720p
INSERT 17 3.425227 Invisible.Target.720p.Bluray.x264-SEPTiC
INSERT 44 2.740964 Paris.Je.T.Aime.2006.720p.BluRay.x264-CiNEFiLE
INSERT 3 5.964769 Bolt [2008] [1080p]
REMOVE_ID 17
REMOVE_ID 12
REMOVE_ID 40
PRINT_IN_ORDER actual/43-test.out
INSERT 53 9.151232 Hitman    720p
INSERT 63 9.584897 G.I.Joe The Rise of Cobra [2009] [720p]
INSERT 62 0.615525 300 [2006]
INSERT 67 6.923587 Rocky 3 [1982] [BluRay] [720p] [AC3]
INSERT 10 4.501124 Doomsday 720p
INSERT 20 0.233403 Cellular.2004.HDTV.720p.AC3.x264-BTT
INSERT 46 2.018515 The.Godfather.Part.I.1972.720p.BluRay.x264-ESiR
REMOVE_ID 10
REMOVE_ID 37
REMOVE_ID 53
PRINT_IN_ORDER actual/43-test.out
INSERT 2 1.990900 Eragon 720p
INSERT 59 5.300373 WALL-E.720p.BluRay.x264-iNFAMOUS
INSERT 28 8.335675 Beauty.And.The.Beast.1991.720p.HDTV.x264-hV
INSERT 41 2.411090 Shrek The Third [2007] [720p]
INSERT 38 7.997828 The Day After Tomorrow 1080p
INSERT 0 0.050758 Hush   720p
INSERT 18 2.157501 James Bond 16 - Licence to Kill [1989]
INSERT 72 2.267329 The Notebook [2004] [1080p]
REMOVE_ID 31
REMOVE_ID 0
REMOVE_ID 19
REMOVE_ID 7
PRINT_IN_ORDER actual/43-test.out
INSERT 47 5.772788 Superman Returns   720p
INSERT 25 4.432990 The Brave One 720P
INSERT 71 8.735117 12 Rounds 720p
INSERT 52 8.861214 Orphan (2009)    1080p
INSERT 22 1.845347 Lord of war   720p
INSERT 15 4.587956 Aladdin [1992] [720p]
INSERT 77 4.706225 Crash    1080p
INSERT 56 5.309156 contents.txt
INSERT 65 5.344133 Fast and the Furious 2 - 1080p
REMOVE_ID 15
REMOVE_ID 3
REMOVE_ID 28
REMOVE_ID 41
PRINT_IN_ORDER actual/43-test.out
INSERT 60 2.905716 X-Men Origin - Wolverine [2009] [1080p]
INSERT 35 0.336766 Saving Private Ryan 720p
INSERT 75 2.969652 Crossing Over 720P
INSERT 6 5.552467 007 Series
INSERT 57 6.069700 Erin Brockovich 2000.720p
INSERT 92 5.964354 Lord Of The Ring  - Return Of The King   720p
INSERT 49 8.054504 James Bond 15 - The Living Daylights [1987] [720p]
INSERT 81 9.024757 Race To Witch Mountain [1080p]
INSERT 94 7.841559 Elite Squad 1080p
INSERT 66 6.632249 Inkheart [2008] [720p]
REMOVE_ID 18
REMOVE_ID 35
REMOVE_ID 46
REMOVE_ID 52
REMOVE_ID 65
PRINT_IN_ORDER actual/43-test.out
INSERT 87 3.873603 Star Wars Saga  Collection 720p
INSERT 83 1.598779 The Final Destination 4 [2009] [720p]
INSERT 93 1.075066 Blood River 720P
INSERT 34 0.515868 Harold & Kumar   720p
INSERT 64 7.257122 Terminator.1984.720p.BluRay.x264-ESiR
INSERT 50 7.367068 Battle for Terra [2007] [1080p]
INSERT 5 0.068397 Night At The Museum 2 [720p]
INSERT 91 8.626480 The Uninvited [2009] [720p]
INSERT 76 0.680592 Red Cliff    1080p
INSERT 88 8.822993 The Tale Of Despereaux [720p]
INSERT 96 2.972061 Final Destination 4   1080p
REMOVE_ID 50
REMOVE_ID 60
REMOVE_ID 91
REMOVE_ID 36
REMOVE_ID 22
PRINT_IN_ORDER actual/43-test.out
INSERT 97 2.322932 James Bond 17 - Golden Eye [1995]
INSERT 8 3.094336 Eastern.Promises.2007.720p
INSERT 105 6.597045 A Knight's Tale [2001] [720p]
INSERT 42 4.822727 Fantastic Four   1080p
INSERT 90 3.725647 Disturbia    720p
INSERT 70 4.591420 The Brave One 720P
INSERT 112 9.323279 Baby on Board (2009)    720p
INSERT 102 1.341444 Stargate-The Ark of Truth 720p
INSERT 98 8.658000 James Bond 03 - GoldFinger [1964]
INSERT 73 3.412676 Ongbak 2003 720p
INSERT 78 7.061405 Street Fighter The Legend Of Chun Li 720P
INSERT 107 0.512666 Big Nothing   720p
REMOVE_ID 9
REMOVE_ID 88
REMOVE_ID 34
REMOVE_ID 97
REMOVE_ID 102
REMOVE_ID 59
PRINT_IN_ORDER actual/43-test.out
INSERT 24 9.909741 Shutter   720p
INSERT 55 9.073276 Bad Boy II [720p]
INSERT 95 1.641497 The Mummy 1-2   1080p
INSERT 103 0.654734 Fast and the Furious 4 - 1080p
INSERT 123 9.091357 Casablana.1942.1080p.BluRay.x264.Codres
INSERT 51 0.929372 Lord Of The Ring  - The Fellowship Of The Ring   720p
INSERT 54 1.309539 Rocky 1 [1976]  [BluRay] [720p] [AC3]
INSERT 69 8.585886 Batman Cartoon - Gotham Knight   720p
INSERT 122 4.143069 Night Train   720p
INSERT 106 7.664104 Cars [2006] [720p]
INSERT 84 7.727290 Inkheart [2008] [720p]
INSERT 13 4.860274 Ong.Bak.2.2008.BluRay.x264-BestHD   720p
INSERT 23 1.144239 Spiderman 3   1080p
REMOVE_ID 55
REMOVE_ID 23
REMOVE_ID 72
REMOVE_ID 78
REMOVE_ID 5
REMOVE_ID 1
PRINT_IN_ORDER actual/43-test.out
