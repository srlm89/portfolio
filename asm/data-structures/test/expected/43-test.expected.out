[ 4 1.019829 Battle for Terra [2007] [1080p] ]
[ 1 4.538350 Shoot.Em.Up.720p.Bluray.x264-SEPTiC ][ 4 1.019829 Battle for Terra [2007] [1080p] ]
[ 1 4.538350 Shoot.Em.Up.720p.Bluray.x264-SEPTiC ][ 21 4.854016 High school musical 2006    720p ][ 32 1.955491 Blade 1   720p ][ 40 9.826899 Robots.WS.AC3.x264.BluRay-PTM   720p ]
[ 1 4.538350 Shoot.Em.Up.720p.Bluray.x264-SEPTiC ][ 12 9.975061 High school musical 2006    720p ][ 21 4.854016 High school musical 2006    720p ][ 27 1.032175 Bride Wars   720p ][ 32 1.955491 Blade 1   720p ][ 40 9.826899 Robots.WS.AC3.x264.BluRay-PTM   720p ]
[ 1 4.538350 Shoot.Em.Up.720p.Bluray.x264-SEPTiC ][ 7 5.316481 Terminator.1984.720p.BluRay.x264-ESiR ][ 9 3.118694 Kill Shot   720p ][ 12 9.975061 High school musical 2006    720p ][ 19 5.367680 Mission Impossible Trilogy   720p ][ 21 4.854016 High school musical 2006    720p ][ 36 8.260447 Hellboy 1    1080p ][ 40 9.826899 Robots.WS.AC3.x264.BluRay-PTM   720p ][ 45 8.391770 Shark Tale [2004] [1080p] ]
[ 1 4.538350 Shoot.Em.Up.720p.Bluray.x264-SEPTiC ][ 3 5.964769 Bolt [2008] [1080p] ][ 7 5.316481 Terminator.1984.720p.BluRay.x264-ESiR ][ 9 3.118694 Kill Shot   720p ][ 19 5.367680 Mission Impossible Trilogy   720p ][ 21 4.854016 High school musical 2006    720p ][ 30 0.589414 Eastern.Promises.2007.720p ][ 31 3.895870 Meet The Fockers    720p ][ 36 8.260447 Hellboy 1    1080p ][ 37 1.272687 SAW IV ][ 44 2.740964 Paris.Je.T.Aime.2006.720p.BluRay.x264-CiNEFiLE ][ 45 8.391770 Shark Tale [2004] [1080p] ]
[ 1 4.538350 Shoot.Em.Up.720p.Bluray.x264-SEPTiC ][ 3 5.964769 Bolt [2008] [1080p] ][ 7 5.316481 Terminator.1984.720p.BluRay.x264-ESiR ][ 9 3.118694 Kill Shot   720p ][ 19 5.367680 Mission Impossible Trilogy   720p ][ 20 0.233403 Cellular.2004.HDTV.720p.AC3.x264-BTT ][ 21 4.854016 High school musical 2006    720p ][ 30 0.589414 Eastern.Promises.2007.720p ][ 31 3.895870 Meet The Fockers    720p ][ 36 8.260447 Hellboy 1    1080p ][ 44 2.740964 Paris.Je.T.Aime.2006.720p.BluRay.x264-CiNEFiLE ][ 45 8.391770 Shark Tale [2004] [1080p] ][ 46 2.018515 The.Godfather.Part.I.1972.720p.BluRay.x264-ESiR ][ 62 0.615525 300 [2006] ][ 63 9.584897 G.I.Joe The Rise of Cobra [2009] [720p] ][ 67 6.923587 Rocky 3 [1982] [BluRay] [720p] [AC3] ]
[ 1 4.538350 Shoot.Em.Up.720p.Bluray.x264-SEPTiC ][ 2 1.990900 Eragon 720p ][ 3 5.964769 Bolt [2008] [1080p] ][ 9 3.118694 Kill Shot   720p ][ 18 2.157501 James Bond 16 - Licence to Kill [1989] ][ 20 0.233403 Cellular.2004.HDTV.720p.AC3.x264-BTT ][ 21 4.854016 High school musical 2006    720p ][ 28 8.335675 Beauty.And.The.Beast.1991.720p.HDTV.x264-hV ][ 30 0.589414 Eastern.Promises.2007.720p ][ 36 8.260447 Hellboy 1    1080p ][ 38 7.997828 The Day After Tomorrow 1080p ][ 41 2.411090 Shrek The Third [2007] [720p] ][ 44 2.740964 Paris.Je.T.Aime.2006.720p.BluRay.x264-CiNEFiLE ][ 45 8.391770 Shark Tale [2004] [1080p] ][ 46 2.018515 The.Godfather.Part.I.1972.720p.BluRay.x264-ESiR ][ 59 5.300373 WALL-E.720p.BluRay.x264-iNFAMOUS ][ 62 0.615525 300 [2006] ][ 63 9.584897 G.I.Joe The Rise of Cobra [2009] [720p] ][ 67 6.923587 Rocky 3 [1982] [BluRay] [720p] [AC3] ][ 72 2.267329 The Notebook [2004] [1080p] ]
[ 1 4.538350 Shoot.Em.Up.720p.Bluray.x264-SEPTiC ][ 2 1.990900 Eragon 720p ][ 9 3.118694 Kill Shot   720p ][ 18 2.157501 James Bond 16 - Licence to Kill [1989] ][ 20 0.233403 Cellular.2004.HDTV.720p.AC3.x264-BTT ][ 21 4.854016 High school musical 2006    720p ][ 22 1.845347 Lord of war   720p ][ 25 4.432990 The Brave One 720P ][ 30 0.589414 Eastern.Promises.2007.720p ][ 36 8.260447 Hellboy 1    1080p ][ 38 7.997828 The Day After Tomorrow 1080p ][ 44 2.740964 Paris.Je.T.Aime.2006.720p.BluRay.x264-CiNEFiLE ][ 45 8.391770 Shark Tale [2004] [1080p] ][ 46 2.018515 The.Godfather.Part.I.1972.720p.BluRay.x264-ESiR ][ 47 5.772788 Superman Returns   720p ][ 52 8.861214 Orphan (2009)    1080p ][ 56 5.309156 contents.txt ][ 59 5.300373 WALL-E.720p.BluRay.x264-iNFAMOUS ][ 62 0.615525 300 [2006] ][ 63 9.584897 G.I.Joe The Rise of Cobra [2009] [720p] ][ 65 5.344133 Fast and the Furious 2 - 1080p ][ 67 6.923587 Rocky 3 [1982] [BluRay] [720p] [AC3] ][ 71 8.735117 12 Rounds 720p ][ 72 2.267329 The Notebook [2004] [1080p] ][ 77 4.706225 Crash    1080p ]
[ 1 4.538350 Shoot.Em.Up.720p.Bluray.x264-SEPTiC ][ 2 1.990900 Eragon 720p ][ 6 5.552467 007 Series ][ 9 3.118694 Kill Shot   720p ][ 20 0.233403 Cellular.2004.HDTV.720p.AC3.x264-BTT ][ 21 4.854016 High school musical 2006    720p ][ 22 1.845347 Lord of war   720p ][ 25 4.432990 The Brave One 720P ][ 30 0.589414 Eastern.Promises.2007.720p ][ 36 8.260447 Hellboy 1    1080p ][ 38 7.997828 The Day After Tomorrow 1080p ][ 44 2.740964 Paris.Je.T.Aime.2006.720p.BluRay.x264-CiNEFiLE ][ 45 8.391770 Shark Tale [2004] [1080p] ][ 47 5.772788 Superman Returns   720p ][ 49 8.054504 James Bond 15 - The Living Daylights [1987] [720p] ][ 56 5.309156 contents.txt ][ 57 6.069700 Erin Brockovich 2000.720p ][ 59 5.300373 WALL-E.720p.BluRay.x264-iNFAMOUS ][ 60 2.905716 X-Men Origin - Wolverine [2009] [1080p] ][ 62 0.615525 300 [2006] ][ 63 9.584897 G.I.Joe The Rise of Cobra [2009] [720p] ][ 66 6.632249 Inkheart [2008] [720p] ][ 67 6.923587 Rocky 3 [1982] [BluRay] [720p] [AC3] ][ 71 8.735117 12 Rounds 720p ][ 72 2.267329 The Notebook [2004] [1080p] ][ 75 2.969652 Crossing Over 720P ][ 77 4.706225 Crash    1080p ][ 81 9.024757 Race To Witch Mountain [1080p] ][ 92 5.964354 Lord Of The Ring  - Return Of The King   720p ][ 94 7.841559 Elite Squad 1080p ]
[ 1 4.538350 Shoot.Em.Up.720p.Bluray.x264-SEPTiC ][ 2 1.990900 Eragon 720p ][ 5 0.068397 Night At The Museum 2 [720p] ][ 6 5.552467 007 Series ][ 9 3.118694 Kill Shot   720p ][ 20 0.233403 Cellular.2004.HDTV.720p.AC3.x264-BTT ][ 21 4.854016 High school musical 2006    720p ][ 25 4.432990 The Brave One 720P ][ 30 0.589414 Eastern.Promises.2007.720p ][ 34 0.515868 Harold & Kumar   720p ][ 38 7.997828 The Day After Tomorrow 1080p ][ 44 2.740964 Paris.Je.T.Aime.2006.720p.BluRay.x264-CiNEFiLE ][ 45 8.391770 Shark Tale [2004] [1080p] ][ 47 5.772788 Superman Returns   720p ][ 49 8.054504 James Bond 15 - The Living Daylights [1987] [720p] ][ 56 5.309156 contents.txt ][ 57 6.069700 Erin Brockovich 2000.720p ][ 59 5.300373 WALL-E.720p.BluRay.x264-iNFAMOUS ][ 62 0.615525 300 [2006] ][ 63 9.584897 G.I.Joe The Rise of Cobra [2009] [720p] ][ 64 7.257122 Terminator.1984.720p.BluRay.x264-ESiR ][ 66 6.632249 Inkheart [2008] [720p] ][ 67 6.923587 Rocky 3 [1982] [BluRay] [720p] [AC3] ][ 71 8.735117 12 Rounds 720p ][ 72 2.267329 The Notebook [2004] [1080p] ][ 75 2.969652 Crossing Over 720P ][ 76 0.680592 Red Cliff    1080p ][ 77 4.706225 Crash    1080p ][ 81 9.024757 Race To Witch Mountain [1080p] ][ 83 1.598779 The Final Destination 4 [2009] [720p] ][ 87 3.873603 Star Wars Saga  Collection 720p ][ 88 8.822993 The Tale Of Despereaux [720p] ][ 92 5.964354 Lord Of The Ring  - Return Of The King   720p ][ 93 1.075066 Blood River 720P ][ 94 7.841559 Elite Squad 1080p ][ 96 2.972061 Final Destination 4   1080p ]
[ 1 4.538350 Shoot.Em.Up.720p.Bluray.x264-SEPTiC ][ 2 1.990900 Eragon 720p ][ 5 0.068397 Night At The Museum 2 [720p] ][ 6 5.552467 007 Series ][ 8 3.094336 Eastern.Promises.2007.720p ][ 20 0.233403 Cellular.2004.HDTV.720p.AC3.x264-BTT ][ 21 4.854016 High school musical 2006    720p ][ 25 4.432990 The Brave One 720P ][ 30 0.589414 Eastern.Promises.2007.720p ][ 38 7.997828 The Day After Tomorrow 1080p ][ 42 4.822727 Fantastic Four   1080p ][ 44 2.740964 Paris.Je.T.Aime.2006.720p.BluRay.x264-CiNEFiLE ][ 45 8.391770 Shark Tale [2004] [1080p] ][ 47 5.772788 Superman Returns   720p ][ 49 8.054504 James Bond 15 - The Living Daylights [1987] [720p] ][ 56 5.309156 contents.txt ][ 57 6.069700 Erin Brockovich 2000.720p ][ 62 0.615525 300 [2006] ][ 63 9.584897 G.I.Joe The Rise of Cobra [2009] [720p] ][ 64 7.257122 Terminator.1984.720p.BluRay.x264-ESiR ][ 66 6.632249 Inkheart [2008] [720p] ][ 67 6.923587 Rocky 3 [1982] [BluRay] [720p] [AC3] ][ 70 4.591420 The Brave One 720P ][ 71 8.735117 12 Rounds 720p ][ 72 2.267329 The Notebook [2004] [1080p] ][ 73 3.412676 Ongbak 2003 720p ][ 75 2.969652 Crossing Over 720P ][ 76 0.680592 Red Cliff    1080p ][ 77 4.706225 Crash    1080p ][ 78 7.061405 Street Fighter The Legend Of Chun Li 720P ][ 81 9.024757 Race To Witch Mountain [1080p] ][ 83 1.598779 The Final Destination 4 [2009] [720p] ][ 87 3.873603 Star Wars Saga  Collection 720p ][ 90 3.725647 Disturbia    720p ][ 92 5.964354 Lord Of The Ring  - Return Of The King   720p ][ 93 1.075066 Blood River 720P ][ 94 7.841559 Elite Squad 1080p ][ 96 2.972061 Final Destination 4   1080p ][ 98 8.658000 James Bond 03 - GoldFinger [1964] ][ 105 6.597045 A Knight's Tale [2001] [720p] ][ 107 0.512666 Big Nothing   720p ][ 112 9.323279 Baby on Board (2009)    720p ]
[ 2 1.990900 Eragon 720p ][ 6 5.552467 007 Series ][ 8 3.094336 Eastern.Promises.2007.720p ][ 13 4.860274 Ong.Bak.2.2008.BluRay.x264-BestHD   720p ][ 20 0.233403 Cellular.2004.HDTV.720p.AC3.x264-BTT ][ 21 4.854016 High school musical 2006    720p ][ 24 9.909741 Shutter   720p ][ 25 4.432990 The Brave One 720P ][ 30 0.589414 Eastern.Promises.2007.720p ][ 38 7.997828 The Day After Tomorrow 1080p ][ 42 4.822727 Fantastic Four   1080p ][ 44 2.740964 Paris.Je.T.Aime.2006.720p.BluRay.x264-CiNEFiLE ][ 45 8.391770 Shark Tale [2004] [1080p] ][ 47 5.772788 Superman Returns   720p ][ 49 8.054504 James Bond 15 - The Living Daylights [1987] [720p] ][ 51 0.929372 Lord Of The Ring  - The Fellowship Of The Ring   720p ][ 54 1.309539 Rocky 1 [1976]  [BluRay] [720p] [AC3] ][ 56 5.309156 contents.txt ][ 57 6.069700 Erin Brockovich 2000.720p ][ 62 0.615525 300 [2006] ][ 63 9.584897 G.I.Joe The Rise of Cobra [2009] [720p] ][ 64 7.257122 Terminator.1984.720p.BluRay.x264-ESiR ][ 66 6.632249 Inkheart [2008] [720p] ][ 67 6.923587 Rocky 3 [1982] [BluRay] [720p] [AC3] ][ 69 8.585886 Batman Cartoon - Gotham Knight   720p ][ 70 4.591420 The Brave One 720P ][ 71 8.735117 12 Rounds 720p ][ 73 3.412676 Ongbak 2003 720p ][ 75 2.969652 Crossing Over 720P ][ 76 0.680592 Red Cliff    1080p ][ 77 4.706225 Crash    1080p ][ 81 9.024757 Race To Witch Mountain [1080p] ][ 83 1.598779 The Final Destination 4 [2009] [720p] ][ 84 7.727290 Inkheart [2008] [720p] ][ 87 3.873603 Star Wars Saga  Collection 720p ][ 90 3.725647 Disturbia    720p ][ 92 5.964354 Lord Of The Ring  - Return Of The King   720p ][ 93 1.075066 Blood River 720P ][ 94 7.841559 Elite Squad 1080p ][ 95 1.641497 The Mummy 1-2   1080p ][ 96 2.972061 Final Destination 4   1080p ][ 98 8.658000 James Bond 03 - GoldFinger [1964] ][ 103 0.654734 Fast and the Furious 4 - 1080p ][ 105 6.597045 A Knight's Tale [2001] [720p] ][ 106 7.664104 Cars [2006] [720p] ][ 107 0.512666 Big Nothing   720p ][ 112 9.323279 Baby on Board (2009)    720p ][ 122 4.143069 Night Train   720p ][ 123 9.091357 Casablana.1942.1080p.BluRay.x264.Codres ]
